news/2024/2024-01-26-mwc-7k-ro24-summary.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2024: Round of 24 Recap"
+date: 2024-01-26 15:00:00 +0000
+---
+
+The osu!mania 7K World Cup begins! Stay updated as participants start revving up their engines to maintain their momentum along the remainder of the tournament.
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)
+
+Continuing on from the osu! World Cup [recaps](https://osu.ppy.sh/home/news/2023-11-28-osu-world-cup-2023-finals-recap), you can once more expect a summary of the latest and greatest of MWC, made possible by community members just like you! This year, **for the first time ever**, the tournament kicked off with a Round of 24! Due to increased interest from various countries that have both applied and qualified, we have been blessed with an extra round and exciting matches straight away!
+
+Before getting into the highlight matches, let's quickly reflect (or familiarise) ourselves with the Round of 24 mappool.
+
+### Mappool showcase
+
+[**View the VOD here!**](https://www.twitch.tv/videos/2032775000)
+
+[**Download the pool here! (50 MB)**](https://packs.ppy.sh/P233%20-%20osu!mania%207K%20World%20Cup%202024%3A%20Round%20of%2024.zip)
+
+With a new tournament comes a suite of new custom beatmaps for everyone to enjoy! This time around, the mapping team is making active efforts to ensure some of these picks are available to be Ranked after the tournament ends. Stay tuned for all [osu! originals](/wiki/osu!_originals) that are up to come — you may see them Ranked earlier than you'd think so!
```

Comment:
Last sentence was moreso meant to infer that these picks are actively being worked on so they can be ranked ASAP. Meant to excite readers rather than re-stating what has been said before. How about something like:
```suggestion
With a new tournament comes a suite of new custom beatmaps for everyone to enjoy! This time around, the mapping team is making active efforts to ensure some of these picks are available to be ranked after the tournament ends. Stay tuned for all [osu! originals](/wiki/osu!_originals) that are up to come — you'll be able to play them in no time!
```

Otherwise, we ditch the last part as suggested.

Reply:
```suggestion
With a new tournament comes a suite of new custom beatmaps for everyone to enjoy! This time around, the mapping team is making active efforts to ensure some of these picks are available to be ranked after the tournament ends. Stay tuned for all [osu! originals](/wiki/osu!_originals) that are up to come — soon right in your beatmap collection!
```